apps/meteor/tests/end-to-end/api/livechat/00-rooms.ts

```diff
@@ -1256,6 +1256,32 @@ describe('LIVECHAT - rooms', () => {
 			await deleteVisitor(visitor.token);
 		});
 
+		it("should set visitor's source as API after uploading a file", async () => {
+			await updateSetting('FileUpload_Enabled', true);
+			await updateSetting('Livechat_fileupload_enabled', true);
+			const visitor = await createVisitor();
+			const room = await createLivechatRoom(visitor.token);
+			await request
+				.post(api(`livechat/upload/${room._id}`))
+				.set(credentials)
+				.set('x-visitor-token', visitor.token)
+				.attach('file', fs.createReadStream(path.join(__dirname, '../../../data/livechat/sample.png')))
+				.expect('Content-Type', 'application/json')
+				.expect(200);
+
+			const { body } = await request
+				.get(api('livechat/visitors.info'))
+				.query({ visitorId: visitor._id })
+				.set(credentials)
+				.expect('Content-Type', 'application/json')
+				.expect(200);
+			expect(body).to.have.property('visitor').and.to.be.an('object');
+			expect(body.visitor).to.have.property('source').and.to.be.an('object');
+			expect(body.visitor.source).to.have.property('type', 'api');
+
+			await deleteVisitor(visitor.token);
```

Comment:
Move to an after

Reply:
I've just moved this to an `afterEach` since in this describe specifically we're already deleting visitors in multiple tests